### PR TITLE
EOS-14632: create bootstrap script for isolated environment

### PR DESCRIPTION
**Pull request by SomwanshiAnjali**

### CODE REVIEW

**Review by ypise — APPROVED** [review]

File: cli/src/cortx_prep.sh [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+[$repo]
+baseurl=file:///tmp/iso_mount/${repo}
+gpgcheck=0
+name=Repository ${repo}
+enabled=1
+
+EOF
+      done
+        echo "INFO: Installing cortx-prvsnr packages" 2>&1 | tee -a ${LOG_FILE}
+        yum install python36-m2crypto salt salt-master salt-minion python36-cortx-prvsnr -y 2>&1 | tee -a ${LOG_FILE}
```
Comment (ypise) [inline cli/src/cortx_prep.sh:58]: Please modify this to:
```
yum install -y \
python36-m2crypto \
salt salt-master salt-minion \
python36-cortx-prvsnr \
2>&1 | tee -a ${LOG_FILE}
```

File: cli/src/cortx_prep.sh [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        prvsnr_version=$(provisioner --version)
+        if [[ $?==0 ]]; then
+          echo "INFO: Successfully installed cortx-prvsnr!!" 2>&1 | tee -a ${LOG_FILE}
+        else
+          echo "ERROR: cortx-prvsnr not installed" 2>&1 | tee -a ${LOG_FILE}
+          exit 1
+        fi
+
+        echo "INFO: Removing /etc/yum.repos.d/bootstrap.repo" 2>&1 | tee -a ${LOG_FILE}
+        rm -rf /etc/yum.repos.d/bootstrap.repo
```
Comment (ypise) [inline cli/src/cortx_prep.sh:70]: Update to:
`rm -f /etc/yum.repos.d/bootstrap.repo`

`-r` is risky

**Review by andkononykhin2 — APPROVED** [review]
@SomwanshiAnjali Looks good. Thank you!

File: cli/src/cortx_prep.sh [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        echo "ERROR: CORTX Release ISO not found at /root/iso/" 2>&1 | tee -a ${LOG_FILE}
+        exit 1
+    fi
+}
+
+function usage {
+  echo "\
+Usage: $0
+
+Installs cortx-prvsnr
```
Comment (ypise) [inline cli/src/cortx_prep.sh:87]: Installs cortx-prvsnr API  (python36-cortx-prvsnr)

File: cli/src/cortx_prep.sh [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+Must be run from primary node.
```
Comment (ypise) [inline cli/src/cortx_prep.sh:89]: Must preferably be run from primary node.

**Review by ypise — CHANGES_REQUESTED** [review]

File: cli/src/cortx_prep.sh [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+}
+trap trap_handler ERR
+
+function install_prvsnr() {
+
+    echo "INFO: Creating a directory to serve as the mount point" 2>&1 | tee -a ${LOG_FILE}
+    mkdir -p /tmp/iso_mount
+
+    if [[ `find /root/iso -name *.iso` ]]; then
+        cortx_iso=$(ls -t /root/iso/cortx-1.0-*-single.iso | head -1 | xargs basename)
```
Comment (ipoddubnyy) [inline cli/src/cortx_prep.sh]: make this a bit more universal, e.g. `cortx-*.iso`. this will help us in the future.

File: cli/src/cortx_prep.sh [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+}
+trap trap_handler ERR
+
+function install_prvsnr() {
+
+    echo "INFO: Creating a directory to serve as the mount point" 2>&1 | tee -a ${LOG_FILE}
+    mkdir -p /tmp/iso_mount/{cortx,cortx-os}
+
+    if [[ `find /opt/isos -name *.iso` ]]; then
+        cortx_iso=$(ls -t /opt/isos/cortx-1.0-*-single.iso | head -1 | xargs basename)
```
Comment (ipoddubnyy) [inline cli/src/cortx_prep.sh:40]: one minor suggestion - make it a bit more universal. something like this: `cortx-*-single.iso`

**Review by ipoddubnyy — APPROVED** [review]
approved with minor improvement

**Review by ypise — APPROVED** [review]

**Pull request merged**